README.md

```diff
@@ -89,4 +84,22 @@ mkvirtualenv -p /usr/local/bin/python3 notifications-template-preview
 pip install -r requirements.txt
 ```
 
-Then to run, activate the virtualenv and call `./scripts/run_app.sh 6013`
+Then create a `version.py` file under the app folder, you can copy `version.py.dist` to `version.py` to get it running.
```

Comment:
ok, but it's still useful to just let people know what needs to be done to run it locally, could change the documentation to use make _generate-version-file instead